tendrl/ceph_integration/ceph.py

```diff
@@ -867,7 +867,7 @@ def heartbeat(fsid=None):
         import rados
     except ImportError:
         # Ceph isn't installed, report no services or clusters
-        return None, {}
+        raise ImportError
```

Comment:
remove the entire try/except  and keep the statement `import rados`, thats all required in this PR